vane-trifles/src/main/java/org/oddlama/vane/trifles/FastWalkingListener.java

```diff
@@ -45,6 +58,15 @@ public void on_player_move(final PlayerMoveEvent event) {
 	public void on_entity_move(final EntityMoveEvent event) {
 		final var entity = event.getEntity();
 
+		// Cancel event if speedwalking is only enabled for players
+		if(players_only_speedwalk) return;
+
+		// Cancel event if speedwalking is disabled for Hostile mobs
+		if(entity instanceof Monster && !hostile_speedwalk) return;
+		
+		// Cancel event if speedwalking is disabled for Villiagers
+		if(entity.getType() == EntityType.VILLAGER && !villiager_speedwalk) return;
+
```

Comment:
@oddlama @Serial-ATA here are the changes I settled on.  

Do you guys like hostile_speedwalk off by default?

Reply:
Yeah, hostile speedwalking was pretty annoying to deal with.